crates/eips/src/eip7702/auth_list.rs

```diff
@@ -20,7 +20,7 @@ pub struct Authorization {
     /// The address of the authorization.
     pub address: Address,
     /// The nonce for the authorization.
-    pub nonce: OptionalNonce,
+    pub nonce: u64,
```

Comment:
we should serde serialize this with alloy_serde::quantity

Reply:
10ce80a